apis/v1beta1/gateway_types.go

```diff
@@ -907,3 +876,22 @@ const (
 	// online and ready to accept client traffic.
 	ListenerReasonPending ListenerConditionReason = "Pending"
 )
+
+const (
+	// "Ready" is a reserved condition type for future use. It should not be used by implementations.
```

Comment:
```suggestion
	// "Ready" is a condition type reserved for future use. It should not be used by implementations.
```

Reply:
Changed as part of https://github.com/kubernetes-sigs/gateway-api/pull/1934 :)